### PR TITLE
Add comments to frustum query proto definition

### PR DESCRIPTION
**Pull request by ksavinash9**
Add comments to frustum query proto definition

### CODE REVIEW

**Comment by ksavinash9** [comment]
@SirVer @thomasschiwietz 
Please, review and merge.

File: point_viewer_grpc_proto_rust/src/proto.proto [hunk 1]
```diff
@@ -51,11 +51,22 @@ message GetPointsInBoxRequest {
 }
 
 message GetPointsInFrustumRequest {
+  // Unit vector defining the translation.
```
Comment (SirVer) [inline point_viewer_grpc_proto_rust/src/proto.proto]: This is not a unit vector.

File: point_viewer_grpc_proto_rust/src/proto.proto [hunk 2]
```diff
@@ -51,11 +51,22 @@ message GetPointsInBoxRequest {
...
   point_viewer.proto.Vector3f translation = 1;
+
+  // Unit vector specifying the rotation quaternion.
```
Comment (SirVer) [inline point_viewer_grpc_proto_rust/src/proto.proto]: Unit quaternion speci....

File: point_viewer_grpc_proto_rust/src/proto.proto [hunk 3]
```diff
@@ -51,11 +51,22 @@ message GetPointsInBoxRequest {
...
   point_viewer.proto.Vector4f rotation = 2;
+
+  // Field of view angle in radians.
   double fovy_rad = 3;
+
+  // Aspect ratio of width to y height.
```
Comment (SirVer) [inline point_viewer_grpc_proto_rust/src/proto.proto]: Is this x / y or y / x?

**Comment by ksavinash9** [comment]
@SirVer Addressed the comments. Please review.

TODO:
I'll update Vector4f to Quaternion in next pull request.

File: point_viewer_grpc_proto_rust/src/proto.proto [hunk 4]
```diff
@@ -51,11 +51,22 @@ message GetPointsInBoxRequest {
...
+  // Vector defining the translation.
   point_viewer.proto.Vector3f translation = 1;
+
+  // Quaternion specifying the rotation.
   point_viewer.proto.Vector4f rotation = 2;
+
+  // Field of view angle in radians.
   double fovy_rad = 3;
+
+  // Aspect ratio of x (width) to y (height).
```
Comment (SirVer) [inline point_viewer_grpc_proto_rust/src/proto.proto]: Suggestion: `// Aspect ratio, i.e. width / height.`

**Comment by ksavinash9** [comment]
@SirVer please, review and merge.

**Comment by SirVer** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user SirVer. Merge queue now has a length of 1.

**Pull request merged**